includes/wf_crm_webform_base.inc

```diff
@@ -57,10 +57,10 @@ abstract class wf_crm_webform_base {
       case 'tax_rate':
         $taxSettings = CRM_Core_BAO_Setting::getItem(CRM_Core_BAO_Setting::CONTRIBUTE_PREFERENCES_NAME, 'contribution_invoice_settings');
         if (is_array($taxSettings) && !empty($taxSettings['invoicing'])) {
-          if ($this->contribution_page) {
+          if ($this->_contribution_page) {
```

Comment:
That's not a typo. See comment above the variable.

Reply:
Ok, removed that bit.  This PHP notice fix should be good though